### PR TITLE
cloud: add the reminder message of charging policy for backups when customers delete dedicated clusters

### PR DESCRIPTION
**Pull request by nicole-yc**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

In current cluster deleting panel, there's no message to remind customers that backups will generate the charging fee , which make customers surprised when checking the bills. So this time, we redesign the UX to remind the customers of the charging policy of automatic and manual backups after deleting clusters.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/delete-tidb-cluster.md [hunk 1]
```diff
@@ -17,19 +17,23 @@ You can delete a cluster at any time by performing the following steps:
     > Alternatively, you can also click the name of the target cluster to go to its overview page, and then click **...** in the upper-right corner.
 
 3. Click **Delete** in the drop-down menu.
-4. In the cluster deleting window, enter your `<organization name>/<project name>/<cluster name>`.
+4. In the cluster deleting window, the prompt message will inform you of the number of backups and the charging policy for backups if you have at least one manual or automatic backups. If you click **Continue** or if no backups are present, you will be prompted to enter `<organization name>/<project name>/<cluster name>` to confirm the deletion.
```
Comment (lilin90) [inline tidb-cloud/delete-tidb-cluster.md:20]:
```suggestion
4. In the cluster deleting window, confirm the deletion.

    - If you have at least one manual or automatic backup, you can see the number of backups and the charging policy for backups. Click **Continue** and enter `<organization name>/<project name>/<cluster name>`.
    - If you do not have any backups, just enter `<organization name>/<project name>/<cluster name>`.
```

File: tidb-cloud/delete-tidb-cluster.md [hunk 2]
```diff
@@ -17,19 +17,23 @@ You can delete a cluster at any time by performing the following steps:
...
 
-5. Click **I understand the consequences. Delete this cluster**.
+5. Click **I understand, delete it**.
 
  Once a backed up TiDB Dedicated cluster is deleted, the existing backup files of the cluster are moved to the recycle bin.
 
-- For backup files from an automatic backup, the recycle bin can retain them for 7 days.
-- For backup files from a manual backup, there is no expiration date.
+- Automatic backups will expire and be automatically deleted once the retention period has passed. The default retention period is 7 days if you don't modify it. 
+- Manual backups will be kept in the Recycle Bin until manually deleted.
```
Comment (lilin90) [inline tidb-cloud/delete-tidb-cluster.md:33]:
```suggestion
- Automatic backups will stop and be automatically deleted once the retention period expires. The default retention period is 7 days if you don't modify it.
- Manual backups will be kept in the Recycle Bin until manually deleted.
```

Comment (nicole-yc) [reply tidb-cloud/delete-tidb-cluster.md:36]: Thanks for comment, but I think stop is not accurate for automatic backup. It's not stop backup but backup files will be expired when the retention time arrived.

File: tidb-cloud/delete-tidb-cluster.md [hunk 3]
```diff
@@ -17,19 +17,26 @@ You can delete a cluster at any time by performing the following steps:
...
+
+  > **Note:**
+  >
+  > Please be aware that backups will continue to incur charges until deleted.
```
Comment (hfxsd) [inline tidb-cloud/delete-tidb-cluster.md:40]:
```suggestion
  > Backups will continue to incur charges until deleted.
```

File: tidb-cloud/delete-tidb-cluster.md (hunk 2 above)
Comment (lilin90) [reply tidb-cloud/delete-tidb-cluster.md:36]: Got the meaning. Then let's keep "expire".

File: tidb-cloud/delete-tidb-cluster.md (hunk 3 above)
Comment (lilin90) [reply tidb-cloud/delete-tidb-cluster.md:40]: The original sentence is exactly the one that is displayed on the TiDB Cloud console page. So I think it's ok to keep it as it is?

**Review by Yuiham — APPROVED** [review]
lgtm

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**